app/views/posts/index.html.erb

```diff
@@ -1,9 +1,42 @@
-<div>
-     <% @posts.each do |post|%>
-    <ul>
-        <li><%= post.title%></li>
-        <li><%= post.text%></li>
-        <li><%= post.comments_counter%></li>
-    </ul>
-    <%end%>
-</div>
\ No newline at end of file
+ <main>
+ <div class=container>  
+ <div class="user-show-info-container">
+   <div class='photo'>
+ <img image_tag ="<%= @user.photo%>", style='height:50px; width: 40px ;margin-bottom:20px;' >
+   </div>
+   <div class='info'>
+      <h1><%=@user.name%></h1>
+     <p><strong>Number of Posts:</strong><span> <%= @user.posts_counter%></span></p>
+   </div>
+</div>
+   <div class="posts-container">
+      <% @posts.each do |post|%>
+      <div>
+        <a href=<%= "posts/#{post.id}" %>>
+      <h1><%= post.title%></h1>
+      </a>
+          <p><%= post.text%></p>
+          <div class="likes-comments">
+            <p>Likes: <%= post.likes_counter%>, </p>
+            <p>Comments: <%= post.comments_counter%></p>
+          </div>  
+      </div>
+        
+      <%end%>
+   </div>
+     <div class="comments">
+          <% if @comments.length>=1 %>
+          <% @comments.each do |comment| %>
+            <div class="comment">
+              <p>Comments:</p>
+              <p><%= comment[:user] %>: <%= comment[:comment] %></p>
+            </div>
```

Comment:
- [ ] Well done implementing some of the changes requested by the previous reviewer, But please make sure that the rendering of your recent comments on a post match the given wireframe. You can see the `<p>Comments:</p>` is continuously rendering on the page whenever a comment is created since it is inside the loop here and this is not making your design match the given wireframe. :blush: 
  
  ### Your Wireframe
  ![image](https://github.com/IsmailChangezi/blog-app-new/assets/92303891/b956f88e-3c6a-44fb-978e-572fe1d7c05c)
  
  ### Given wireframe
  ![image](https://github.com/IsmailChangezi/blog-app-new/assets/92303891/bec9fe08-b0a4-4b75-808f-e71878e24e40)